libzmix/bulletproofs_amcl/README.md

```diff
@@ -0,0 +1,5 @@
+# Bulletproofs
+
+1. On a pairing friendly curve. BLS12-381 and BN254 for now. Change value of default feature to change the curve,
```

Comment:
Can you please reword this? I think you are conveying that to change that the implementation provides two curve choices that can be selected in cargo.toml.

Reply:
Ok